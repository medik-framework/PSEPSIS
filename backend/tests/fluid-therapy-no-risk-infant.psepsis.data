{
  "Normal Saline Count": 0,
  "Lactated Ringer Count": 0
}
