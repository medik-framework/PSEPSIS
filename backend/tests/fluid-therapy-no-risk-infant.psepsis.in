[
  {
    "id": "tabletApp",
    "action": "broadcast",
    "eventName": "StartScreening",
    "eventArgs": []
  },
  {
    "id": "tabletApp",
    "action": "broadcast",
    "eventName": "AgeEntered",
    "eventArgs": [3],
    "waitFor":
     {
       "name": "Instruct",
       "args": ["get age"]
     }
  },
  {
    "id": "tabletApp",
    "action": "broadcast",
    "eventName": "StartFluidTherapy",
    "eventArgs": [],
    "waitFor":
     {
       "name": "Instruct",
       "args": ["get weight"]
     }
  },
  {
    "id": "tabletApp",
    "action": "broadcast",
    "eventName": "FluidOverloadRisksEntered",
    "eventArgs": [0],
    "waitFor" : {
      "name": "Instruct",
      "args": ["get fluid overload risks"]
    }
  },
  {
    "action": "exit",
    "waitFor": {
      "name": "Recommend"
    }
  }
]
