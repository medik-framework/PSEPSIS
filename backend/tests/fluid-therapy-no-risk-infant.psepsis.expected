[
  {
    "id": "tabletApp",
    "interface": "TabletApp",
    "name": "Instruct",
    "args": [
      "get age"
    ]
  },
  {
    "id": "tabletApp",
    "interface": "TabletApp",
    "name": "ConfirmAgeEntered",
    "args": []
  },
  {
    "id": "tabletApp",
    "interface": "TabletApp",
    "name": "Instruct",
    "args": [
      "get weight"
    ]
  },
  {
    "id": "tabletApp",
    "interface": "TabletApp",
    "name": "Instruct",
    "args": ["get fluid overload risks"]
  },
  {
    "id": "tabletApp",
    "interface": "TabletApp",
    "name": "ConfirmFluidOverloadRisksEntered",
    "args": []
  },
  {
    "id": "datastore",
    "interface": "Datastore",
    "name": "Obtain",
    "args": ["Normal Saline Count"]
  },
  {
    "id": "datastore",
    "interface": "Datastore",
    "name": "Obtain",
    "args": ["Lactated Ringer Count"]
  },
  {
    "id": "tabletApp",
    "interface": "TabletApp",
    "name": "AlternativeDrugs",
    "args": [
      "Administer 5-10 ml/kg Normal Saline or Lactated Ringer.",
      "Normal Saline",
      "Fluid Therapy",
      "5",
      "Lactated Ringer",
      "Fluid Therapy",
      "5"
    ]
  }
]
